// See README.txt for information and build instructions.

package rpctest;

option java_package = "com.example.rpctest";
option java_outer_classname = "CalculatorProtos";

message CalculatorAddIn {
  required int32 first = 1;
  required int32 second = 2;
}

message CalculatorAddOut {
  required int32 result = 1;
}

message CalculatorSubIn {
  required int32 first = 1;
  required int32 second = 2;
}

message CalculatorSubOut {
  required int32 result = 1;
}

message CalculatorMulIn {
  required int32 first = 1;
  required int32 second = 2;
}

message CalculatorMulOut {
  required int32 result = 1;
}

message CalculatorDivIn {
  required int32 first = 1;
  required int32 second = 2;
}

message CalculatorDivOut {
  required int32 result = 1;
}

message CalculatorGetNumOpsIn {
}

message CalculatorGetNumOpsOut {
  required int32 result = 1;
}

